msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/TestCase832430.java

```diff
@@ -0,0 +1,140 @@
+import com.microsoft.identity.internal.testutils.labutils.LabUserQuery;
+
+import org.junit.Test;
+
+import java.util.concurrent.CountDownLatch;
+
+//Joined AcquireToken test with MSAL and Broker
+//https:identitydivision.visualstudio.com/DevEx/_workitems/edit/832430
+public class TestCase832430 extends AbstractMsalBrokerTest {
+
+    @Test
+    public void test_832430() throws InterruptedException {
+        final String username = mLoginHint;
+        final String password = LabConfig.getCurrentLabConfig().getLabUserPassword();
+
+        //acquiring token
+        final CountDownLatch latch = new CountDownLatch(1);
+
+        final AcquireTokenParameters parameters = new AcquireTokenParameters.Builder()
+                .startAuthorizationFromActivity(mActivity)
+                .withLoginHint(mLoginHint)
+                .withResource(mScopes[0])
+                .withCallback(successfulInteractiveCallback(latch))
+                .withPrompt(Prompt.SELECT_ACCOUNT)
+                .build();
+
+        final InteractiveRequest interactiveRequest = new InteractiveRequest(
+                mApplication,
+                parameters,
+                new OnInteractionRequired() {
+                    @Override
+                    public void handleUserInteraction() {
+                        final PromptHandlerParameters promptHandlerParameters = PromptHandlerParameters.builder()
+                                .prompt(PromptParameter.SELECT_ACCOUNT)
+                                .loginHint(mLoginHint)
+                                .sessionExpected(false)
+                                .consentPageExpected(false)
+                                .speedBumpExpected(false)
+                                .broker(getBroker())
+                                .expectingBrokerAccountChooserActivity(true)
+                                .registerPageExpected(true)
+                                .build();
+
+                        new AadPromptHandler(promptHandlerParameters)
+                                .handlePrompt(username, password);
+                    }
+                }
+        );
+
+        interactiveRequest.execute();
+        latch.await();
+
+        final IAccount account = getAccount();
+
+        //acquiring token silently
+        final CountDownLatch silentLatch = new CountDownLatch(1);
+
+        final AcquireTokenSilentParameters silentParameters = new AcquireTokenSilentParameters.Builder()
+                .forAccount(account)
+                .fromAuthority(account.getAuthority())
+                .forceRefresh(false)
+                .withResource(mScopes[0])
+                .withCallback(successfulSilentCallback(silentLatch))
+                .build();
+
+        mApplication.acquireTokenSilentAsync(silentParameters);
+        silentLatch.await();
+
+        //forwarding time 1 day
+        TestContext.getTestContext().getTestDevice().getSettings().forwardDeviceTimeForOneDay();
+
```

Comment:
Let's add a Thread.sleep for 30 seconds here to expire the in-memory throttling cache

Reply:
added thread.sleep for 30 seconds.